modules/core/include/visp3/core/vpCameraParameters.h

```diff
@@ -333,6 +360,8 @@ class VISP_EXPORT vpCameraParameters
   inline double get_v0() const { return v0; }
   inline double get_kud() const { return kud; }
   inline double get_kdu() const { return kdu; }
+  inline std::vector<double> get_distortion_coefs() const { return m_dist_coefs; }
```

Comment:
There could be a confusion with the pine hole `kud` and `kdu` coefficients. I suggest to rename `get_distortion_coefs()` into `getKannalaBrandtDistortionCoeficients()`